src/main/java/org/sagebionetworks/web/server/servlet/filter/CrawlFilter.java

```diff
@@ -470,8 +359,12 @@ private String getEntityHtml(String entityId)
       request.setNextPageToken(childList.getNextPageToken());
       i++;
     } while (i < MAX_CHILD_PAGES && childList.getNextPageToken() != null);
-    html.append("</body></html>");
-    return html.toString();
+    response.setBody(html.toString());
+
+    // SWC-6609: removed for now
+    //    response.head = getDatasetScriptElement(bundle, plainTextWiki);
```

Comment:
Still not returning the dataset json ld script data.  We could turn this on, but I'd rather wait for a backend solution, and I don't want us to become very popular in Google Dataset Search before we implement managed plans